### PR TITLE
[Term Entry] Java Calendar getActualMaximum()

### PR DESCRIPTION
**Pull request by Erick559**
### Description

I added the .getActualMaximum to the Java Calendar class. The markdown file contains a brief description of the method and an example showing what it does. I am currently doing the 'Learn Git and GitHub' short course and contributing to this repository was one of the tasks needed.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+Title: '.getActualMaximum()'
+Description: 'A Calendar class method that returns the maximum value the specified calendar field could have, based on the time value of the calendar'
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:3]:
```suggestion
Description: 'Returns the maximum value the given calendar field can have, conditional on the time value of the calendar.'
```

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+The **`.getActualMaximum()`** method of the `Calendar` class retrieves the maximum value the specified calendar field could have, based on the time value of the calendar.
+
+## Syntax
+
+```pseudo
+calendar.getActualMaximum(int field)
+```
+
+## Parameters
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:23]:
```suggestion
```

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+`int field`: The calendar field the maximum value is to be retrieved from. For example `Calendar.MONTH` and `calendar.YEAR`
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:25]:
```suggestion
The `getActualMaximum` method takes only one parameter, `int field`. This is the calendar field the maximum value is to be retrieved from. For example `Calendar.MONTH` and `calendar.YEAR`
```

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Example
+
+In the following example, `.getActualMax()` is used to retrieve the maximum value of both the `Calendar.MONTH` and `Calendar.YEAR` fields.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:29]:
```suggestion
In the following example, `.getActualMaximum()` is used to retrieve the maximum value of both the `Calendar.MONTH` and `Calendar.YEAR` fields.
```

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+            system.out.println("Maximum year is: " + year);
+
+        //Retrieving the maximum value the month field can have 
+
+        int month = 
+            calendar.getActualMaximum(Calendar.MONTH);
+            system.out.println("Maximum MONTH is: " + month); 
+
+   }
+}
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:52]:
```suggestion
import java.util.Calendar;

 // CalendarMaxDemo.java
 public class CalendarMaxDemo {              
  public static void main(String[] args) {

  // Creating a calendar instance 
  Calendar calendar = Calendar.getInstance();

  // Retrieving the maximum value  the year field  can have   
  int year =  calendar.getActualMaximum(Calendar.YEAR);
  System.out.println("Maximum year is: " + year);

  // Retrieving the maximum value the month field can have 
    
  int month = calendar.getActualMaximum(Calendar.MONTH);
  System.out.println("Maximum MONTH is: " + month); 
  }
}
```

Comment (caupolicandiaz) [reply content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:52]: There should always be one space before the start of a comment and docs code blocks must use 2 spaces, not tabs

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md (hunk 1 above)
Comment (caupolicandiaz) [reply content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md:3]: The original description and example all appear to be directly copied from https://www.geeksforgeeks.org/calendar-getactualmaximum-method-in-java-with-examples/

All submitted entries must be original content. Copying content from another site is plagiarism, and is not permitted in docs. Please revise this entry and example to replace all copied content.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Erick559 I've added my initial comments. Please review carefully and make the necessary revisions. Tag me when you've finished making changes. Thanks

File: content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md (hunk 5 above)
Comment (Erick559) [reply content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md]: @caupolicandiaz Thank you for your insight. I reviewed my entry and made changes in accordance to yout comments. I changed the descpription of the method as it was truly identical to the geekforgeeks one. I also created a new example for the entry. Thanks

Comment (Erick559) [reply content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md]: Also i don't know why my commit has 2 authors yet I commited. Could you please help me out. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@Erick559 Just made a couple of direct edits to correct the example and the syntax section. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@Erick559 Made one minor edit and it looks good to merge.

**Pull request merged**